### PR TITLE
docs: add documentation for setup kubernetes cluster

### PR DESCRIPTION
**Pull request by Javlopez**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,3 +3,11 @@
 For managing the infrastructure to run our demo system
 
 A new best practice for managing K8s infrastructure is to store configuration in Git and use PRs to make sure changes get reviewed. Seems like a great idea.
+
+## Setup a Kubernetes Cluster for KSD using Hetzner Cloud
+KSD can be effortlessly deployed in different environments, such as bare metal or cloud.
```
Comment (galexrt) [inline README.md]: Please add new lines after each header and before a list starts

```suggestion
## Setup a Kubernetes Cluster for KSD using Hetzner Cloud

KSD can be effortlessly deployed in different environments, such as bare metal or cloud.
```

File: kubernetes-cluster-demo/setup-demo.md [hunk 2]
```diff
@@ -0,0 +1,159 @@
+## Setup a Kubernetes Cluster for KSD using Hetzner Cloud
+
+This guide is to be ready and test KSD easily.
+
+### IMPORTANT:
+In order to be able to follow all the steps you would need a Hetzner Api token, to get this you will need: 
+- Sign in into the Hetzner Cloud Console
```
Comment (galexrt) [inline kubernetes-cluster-demo/setup-demo.md]:
```suggestion
In order to be able to follow all the steps you would need a Hetzner Api token, to get this you will need: 

- Sign in into the Hetzner Cloud Console 
```

**Review by dave-at-koor — APPROVED** [review]
Looks good.

File: kubernetes-cluster-demo/setup-demo.md [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+
+> If you have troubles, please visit the Hetzner Cloud Documents [https://docs.hetzner.cloud/](https://docs.hetzner.cloud/)
+
+## Requirements
+
+- Kubeone
+  - You can install it by using `curl -sfL https://get.kubeone.io | sh`
+- Terraform v1.5.2
+- A new ssh key only for this purpose (how to generate a new ssh-key)
+- Kubectl (how to install kubectl)
```
Comment (zalsader) [inline kubernetes-cluster-demo/setup-demo.md:27]: Are these "how to ... " supposed to be links?

**Pull request merged**